src/awkward/_v2/contents/regulararray.py

```diff
@@ -165,6 +165,69 @@ def _carry(self, carry, allow_lazy, exception):
             self._parameters,
         )
 
+    def compact_offsets64(self, start_at_zero):
+        nplike = self.nplike
+        out = ak._v2.index.Index64.empty(self._length + 1, self.nplike)
+        self._handle_error(
+            nplike["awkward_RegularArray_compact_offsets", out.dtype.type](
+                out.to(nplike),
+                self._length,
+                self._size,
+            )
+        )
+        return out
+
+    def broadcast_tooffsets64(self, offsets):
```

Comment:
Quick request: let's make these (`compact_offsets64` and `broadcast_tooffsets64`) private methods (`_compact_offsets64` and `_broadcast_tooffsets64`). I'm doing the same in #1059.